client/src/components/ForumPost/ForumPost.js

```diff
@@ -53,8 +53,8 @@ export default function ForumPost(props) {
               {" Likes"}
             </p>
             <p>
-              {props.comments}
-              {" Comments"}
+              {props.comments.length}
+              {props.comments.length === 1 ? " Comment" : " Comments"}
```

Comment:
Do we want to render the number of comments like this? Can you think of a better way to do this?

Reply:
@ZakkMan What is the context in regards to your comment?  This render matches the design mockup.  Are you thinking about deviating from the design?  If so, then yes, I would probably put the comment & like counts in their respective buttons.  
If you refer to code, this seems pretty succinct.  Can you provide a little more detail as to potential issues you see?

Reply:
Can you turn this into a single line rather than two?

Reply:
Done